industry_restaurant/demo/pos_order.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_order_6" model="pos.order">
+        <field name="session_id" ref="pos_session_1"/>
+        <field name="partner_id" ref="res_partner_10"/>
+        <field name="employee_id" ref="hr.employee_admin"/>
+        <field name="customer_count">1</field>
+        <field name="currency_rate">1.0</field>
+        <field name="amount_tax">0.6</field>
+        <field name="amount_return">0.0</field>
+        <field name="amount_total">4.6</field>
+        <field name="amount_paid">5.0</field>
+        <field name="pos_reference">Order 00007-007-0003</field>
```

Comment:
pos_reference is not generated in the order creation?

Reply:
we need this field